exchanges/bitstamp/bitstamp_test.go

```diff
@@ -973,29 +987,48 @@ func TestOrderbookZeroBidPrice(t *testing.T) {
 
 func TestGetWithdrawalsHistory(t *testing.T) {
 	t.Parallel()
+
 	sharedtestvalues.SkipTestIfCredentialsUnset(t, b)
-	_, err := b.GetWithdrawalsHistory(context.Background(), currency.BTC, asset.Spot)
-	if err != nil {
-		t.Error(err)
+	h, err := b.GetWithdrawalsHistory(context.Background(), currency.BTC, asset.Spot)
+	assert.NoError(t, err, "GetWithdrawalsHistory should not error")
+	if mockTests {
```

Comment:
This will never be hit because of `SkipTestIfCredentialsUnset`. Needs a `if !mockTests { sharedtestvalues.SkipTestIfCredentialsUnset(t, b)` check

Reply:
Done